src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
```

Comment:
Change the import name from 'Comments' to 'Comment'. The type name should represent a single comment, not a collection of comments.